mix.lock

```diff
@@ -1,6 +1,8 @@
 %{
+  "dialyxir": {:hex, :dialyxir, "1.4.3", "edd0124f358f0b9e95bfe53a9fcf806d615d8f838e2202a9f430d59566b6b53b", [:mix], [{:erlex, ">= 0.2.6", [hex: :erlex, repo: "hexpm", optional: false]}], "hexpm", "bf2cfb75cd5c5006bec30141b131663299c661a864ec7fbbc72dfa557487a986"},
```

Comment:
It seems you forgot to commit the `mix.lock` file.

You can just amend it to your previous commit and I'll merge this :+1: